frontend_v2/pages/newpark.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useState, useEffect, useRef} from 'react'
+import styles from '../styles/Newentry.module.css'
+import ImageUploading from 'react-images-uploading'
+import Camera from 'react-html5-camera-photo'
+import Link from 'next/link'
+import AddParkForm from '../components/AddNewParkForm'
+import { PrismaClient, Tree, Prisma, GreenArea } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+export async function getServerSideProps(){
+    const greenAreas: GreenArea[] = await prisma.greenArea.findMany();
+    
+    return {
+        props: {
+            initialPark: JSON.parse(JSON.stringify(greenAreas))
```

Comment:
You should be able to simply use `initialPark: greenAreas`
"JSON.parse(JSON.stringify())" is redundant